pyproject.toml

```diff
@@ -115,6 +115,7 @@ exclude = [
     "src/puya/_vendor",
     "stubs/algopy\\.py",
 ]
+files = ['stubs/*', 'src/*']
```

Comment:
I think it will better for now to just add algopy_testing as another exclusion rather than modifying files